src/crypto/encryptor.ts

```diff
@@ -0,0 +1,30 @@
+import crypto from "crypto";
+import { Randomness } from "./randomness";
+import { ScryptKeyDerivationParams } from "./derivationParams";
+import { CipherAlgorithm, DigestAlgorithm, Version, KeyDerivationFunction } from "./constants";
+import {EncryptedData} from "./encryptedData";
+
+export class Encryptor {
+  public static encrypt(data: Buffer, password: string, randomness: Randomness = new Randomness()): EncryptedData {
+    const kdParams = new ScryptKeyDerivationParams();
```

Comment:
I confirm the login here is the one from the previous constructor of `UserWallet` ✔️